.github/workflows/utils.py

```diff
@@ -20,11 +20,12 @@ class CustomTemplate(Template):
 
 class Action:
 
-    def __init__(self, name, exclude_list, matrix, step_list):
+    def __init__(self, name, exclude_list, matrix, step_list, strategy_params = od([('fail-fast', 'false')])):
```

Comment:
We can leave it like this, or have a default STRATEGY constant in the gen_github_actions.py file (similar to what we do for EXCLUDE_LIST.

I think I prefer this second option now that I think about it